src/components/articles/SearchResults.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="mt-10 computer:ml-auto computer:mr-10 computer:w-3/5">
+    <TitleSeparator :title="title" />
```

Comment:
```<TitleSeparator :title="title" />
    <div class="rounded-2xl bg-white p-4">
      <div v-if="articles.length === 0" class="mb-5 h-32 rounded-xl bg-white text-center">
        <p class="pt-12 text-2xl font-semibold text-slate-500">There are no articles including this search term.</p>
      </div>
      <div v-if="articles.length" class="mb-5 rounded-xl bg-white text-center">
        <ul>
          <li v-for="(article, index) in articles" :key="index">
            <ArticlePreview :article="article" @showArticle="showArticle" />
          </li>
        </ul>
      </div>
```
This can be separated to a component